src/state/slices/limitOrderInputSlice/selectors.ts

```diff
@@ -81,6 +81,6 @@ export const selectBuyAmountCryptoBaseUnit = createSelector(
       value: buyAmountSellAssetBaseUnit,
       inputExponent: sellAsset.precision,
       outputExponent: buyAsset.precision,
-    }).toFixed()
+    }).toFixed(0)
```

Comment:
Will have to confirm this doesn't re-introduce the precision loss resulting is major differences in submitted amounts vs input

Reply:
Ok confirmed, this has virtually no impact on precision so we're good!